tests/test_sbh_submissions.py

```diff
@@ -1,123 +1,201 @@
+    def test_submit_sub_collection(self):
+        sbh = SynBioHub(SD2Constants.SD2_STAGING_SERVER, self.user, self.password,
+                        SD2Constants.SD2_STAGING_SERVER + '/sparql', SD2Constants.SD2_SERVER)
 
-#           sbolDoc.addComponentDefinition(c)
-#           sbolDoc.addSequence(s)
+        sbol.Config.setOption('sbol_typed_uris', False)
+        sbol.Config.setOption('validate', False)
+        sbol.setHomespace('http://dummy.org')
 
-#       server = SBHConstants.BBN_SERVER
-#       sbhQuery = SynBioHubQuery(server)
-#       displayId = 'stress1_reg'
-#       name = 'dummy_CompDef'
-#       description = 'A collection full of dummy ComponentDefinitions'
-#       version = '1'
-#       sbhQuery.submit_NewCollection(sbolDoc, displayId, name, description, version)
+        doc = sbol.Document()
 
-#   def test_stress2(self):
-#       # write to new collections with different versions
-#       server = SBHConstants.BBN_SERVER
-#       sbhQuery = SynBioHubQuery(server)
+        collection_ID = 'adapter_test_2'
+        collection_name = 'Adapter Test Collection 2'
+        collection_description = 'This is a test of the upload_sbol module of synbiohub_adapter.'
+        collection_version = '1'
 
-#       rule30_sbol = 'examples/rule30-Q0-v2.xml'
-#       description = "same rule of 30 but different versioning collection."
-#       sbolDoc = loadSBOLFile(rule30_sbol)
-#       sbh_connector = sbhQuery.login_SBH()
+        sub_collection_ID = 'sub_adapter_test'
+        sub_collection_name = 'Sub Adapter Test Collection'
+        sub_collection_description = 'This is a test of the upload_sbol module of synbiohub_adapter.'
+        sub_collection_version = '1'
 
-#       sbolDoc.description = description
-
-#       for v in range(0, 3):
-#           version = str(v)
-#           sbolDoc.version = version
-#           sbolDoc.displayId = 'stress2_reg' + version
-#           sbolDoc.name = 'rule30_Q0_v' + version
-
-#           sbhQuery.submit_Collection(sbh_connector, sbolDoc, True, 0)
+        sbh.submit_collection(doc, collection_ID, collection_version, collection_name,
+                              collection_description, 0, sub_collection_ID, sub_collection_version,
+                              sub_collection_name, sub_collection_description, overwrite=True)
 
+        collection_identity = '/'.join([SD2Constants.SD2_SERVER, 'user', self.user, collection_ID,
+                                        collection_ID + '_collection', '1'])
+        sub_collection_identity = '/'.join([SD2Constants.SD2_SERVER, 'user', self.user, collection_ID,
+                                            sub_collection_ID, '1'])
 
-# if __name__ == '__main__':
-#   unittest.main()
+        query_result = sbh.query_collection_members(collection_uris=[collection_identity])
+
+        assert (len(query_result[collection_identity]) == 1 and sub_collection_identity
+                in query_result[collection_identity])
```

Comment:
`assert` is not part of the unittest framework, and is really intended for runtime checks for invariants. It's not what you want in a unit test.

In this case, you really want to use self.assert* methods, like [assertEqual](https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertEqual) and [assertIn](https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertIn).

You should also break this up so it's easier to see which assertion is failing, rather than lumping them all together into single assertion which makes it hard to tell which part is failing.

Here's a rewrite:

```python
	self.assertIn(collection_identity, query_result)
        self.assertEqual(len(query_result[collection_identity]), 1)
        self.assertIn(sub_collection_identity, query_result[collection_identity])
```

The same goes for all the places where you use `assert` instead of `self.assert*`.